Orange/widgets/visualize/owruleviewer.py

```diff
@@ -0,0 +1,251 @@
+import numpy as np
+
+from Orange.data import Table
+from Orange.widgets import widget, gui, settings
+from Orange.widgets.utils.itemmodels import PyTableModel
+
+from PyQt4.QtCore import Qt, QSize, pyqtSignal, QRectF, QLineF, QLocale
+from PyQt4.QtGui import (QApplication, QLabel, QTableView, QStandardItem,
+                         QStandardItemModel, QSortFilterProxyModel,
+                         QMainWindow, QMouseEvent, QGraphicsView,
+                         QPen, QBrush, QColor, QAbstractItemView,
+                         QItemDelegate, QStyledItemDelegate, QPainter,
+                         QFontMetrics, QStyle)
+
+
+class Rule:  # temp - DELETE
+    curr_class_dist = [0, 28, 0]
+
+    def __str__(self):
+        return "IF diau g>=0.172 AND Elu 210>=-0.069 THEN function=Resp"
+rules = []
+
+
+class OWRuleViewer(widget.OWWidget):
+    name = "Rule Viewer"
+    description = "Review rules induced from data."
+    icon = ""
+    priority = 18
+
+    inputs = None
+    outputs = None
+
+    want_basic_layout = True
+    want_main_area = True
+    want_control_area = False
+
+    UserAdviceMessages = [
+        #  widget.Message()
+    ]
+
+    headers = ["IF cond", "", "THEN class", "quality", "class dist"]
+
+    def __init__(self):
+        raise NotImplementedError
+        self.rule_list = None
+
+        self.tableModel = PyTableModel(sequence=[], parent=self, editable=True)
```

Comment:
`editable=True` makes the view respond to double-click so the user can write their own values.